packages/api/db/migrations/20200706102336_create_pbs_table.ts

```diff
@@ -0,0 +1,16 @@
+import Knex from "knex";
+
+export const up = async (knex: Knex) => {
+    return knex.schema.createTable("pbs", table => {
+        table.integer("userid").notNullable();
+        table.integer("rank").notNullable();
```

Comment:
Ok so the thing is we don't need the rank for bs cuz I guess you have this because you want to order them but if it a pb it's clearly bigger than the others and more recent so it's easily sortable.

Reply:
The rank is for ordering the PBs.

Reply:
Well, now that I think about it, it's kind of unnecessary, since you can just sort manually.